### PR TITLE
175636784 setup codeclimate test coverage

### PR DESCRIPTION
**Pull request by bigelio**
#### What does this PR do?
setup codeclimate test coverage
#### Description of Task to be completed?
Connecting our github repo to codeclimate platform
Adding a test coverage badge in README.md file
Adding a maintainability badge in README.md file
#### How should this be manually tested?
run ``git clone https://github.com/atlp-rwanda/tech-strikers-bn-frontend.git``
run `cd tech-strikers-bn-frontend`
run `git checkout ch-setup-codeclimate-test-coverage-175636784`
Check for maintainability badge and test coverage badge in README.md file
#### Any background context you want to provide?
no
#### What are the relevant pivotal tracker stories?
[175636784](https://www.pivotaltracker.com/story/show/175636784)

### CODE REVIEW

**Comment by Duncanwar** [comment]
Don't you have to add codeclimate.yml file?

**Comment by bigelio** [comment]
I found that it may prevent some functionalities to work.
But if you think it is important to add it you can guide me?

**Review by hervebutera — DISMISSED** [review]

**Review by Thiernope — DISMISSED** [review]

File: test/App.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
```
Comment (fabriceumuhire) [inline test/App.test.js]: Are we suppose to have this file twice?

File: test/__snapshots__/App.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (fabriceumuhire) [inline test/__snapshots__/App.test.js.snap]: This one two should be deleted.

**Review by fabriceumuhire — COMMENTED** [review]
Check test files and remove repetitive files.

**Comment by Duncanwar** [comment]
The workflows are not running! Solve the conflicts

**Comment by Duncanwar** [comment]
Solve these conflicts

**Review by tytyne — APPROVED** [review]

**Review by hervebutera — APPROVED** [review]

**Comment by fabriceumuhire** [comment]
Can you fix the code climate issues?

**Pull request merged**